runners/wikiConsole/templates/buildNg.asciidoc

```diff
@@ -0,0 +1,19 @@
+Build an Angular Project with build ng
```

Comment:
I would just write 'Build the Angular project' because the command is described in the text